src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Metadata.cs

```diff
@@ -204,10 +204,9 @@ private static Checksum GetMetadataChecksumSlow(Solution solution, PortableExecu
 
         private static Task<SymbolTreeInfo> CreateMetadataSymbolTreeInfoAsync(
             Solution solution, Checksum checksum,
-            PortableExecutableReference reference,
-            CancellationToken cancellationToken)
+            PortableExecutableReference reference)
         {
-            var creator = new MetadataInfoCreator(solution, checksum, reference, cancellationToken);
+            var creator = new MetadataInfoCreator(solution, checksum, reference);
```

Comment:
i'm more concerned that MetadataInfoCreator doesn't respect cancellation.

Reply:
Would you like me to file a follow-up issue for investigation?

Reply:
Filed #43408